client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -106,12 +106,20 @@ export const SeoForm = React.createClass( {
 	getInitialState() {
 		return {
 			...stateForSite( this.props.site ),
-			seoTitleFormats: {},
-			isRefreshingSiteData: false,
-			dirtyFields: Set()
+			seoTitleFormats: this.props.storedTitleFormats,
+			dirtyFields: Set(),
```

Comment:
Maybe leave a quick comment that you're manually updating this, and that this protects against updates from polling.